### ISSUE TITLE
[Feature Idea]

### ISSUE THREAD

**Issue by willemneal** [near-apps/linkdrop-proxy#2]
We could add another function, `send_with_callback`, which adds an argument for a contract that will be called after the claims process is over.  This will allow other contracts only have to implement a callback but get all the benefits of the linkdrop. The contract could also have it's own send method to make the proxy and it would fit nicely into the existing pipelines.

I'll take a crack at this this weekend.

**Comment by mattlockyer** [comment]
I like this idea.

Just to clarify, `send_with_callback` will save the callback contractId, methodName, args and it will be called when "something" claims the linkdrop?

**Comment by willemneal** [comment]
Yep I have to update tests but here is my working prototype:  #3.

**Comment by willemneal** [comment]
For now I just assume the contract has a callback method with just accound_id, but the change to add a custom method and args should be easy.
